internal/cmd/vet.go

```diff
@@ -299,6 +299,10 @@ func (c *checker) checkSQL(ctx context.Context, s config.SQL) error {
 	req := codeGenRequest(result, combo)
 	cfg := vetConfig(req)
 	for i, query := range req.Queries {
+		if result.Queries[i].Flags[QueryFlagSqlcVetDisable] {
+			fmt.Printf("Skipping vet rules for query %s", query.Name)
```

Comment:
I think printing to stderr is fine and more correct than printing to stdout. Can we change the format to match error output? Also, if you want to suppress the output by default, you can check to see if `debug.Active` is true and only print it then.

```
fmt.Fprintf(c.Stderr, "%s: %s: %s: skipping\n", query.Filename, q.Name, name)
```

Reply:
The problem with printing to stderr I think is that's where "normal" vet rule errors are reported, so skipping a rule would trigger a vet fail every time right?

The `debug.Active` check would suppress that except in cases where someone explicitly opted-in, but I'm still not sure it's the right thing to do.